### ISSUE TITLE
Status info display exception in html files

### ISSUE THREAD

**Issue by wu-clan** [pytest-dev/pytest-html#663]
In 4.0.0rc4, when I run the test and terminate it manually, the html file shows:

```
There are still tests running.

Reload this page to ge the latest results!
```

![image](https://user-images.githubusercontent.com/52145145/234769410-7cbda0b1-f445-4ad4-8d9d-fef03374a79b.png)

Should it show up as an unexpected test abort?

**Comment by wu-clan** [comment]
and manually aborting without printing the html file generation information in the terminal

**Comment by BeyondEvil** [comment]
Yes, we're not handling user initiated aborting and probably won't. The report should display all the tests that have run up to that point.

**Comment by BeyondEvil** [comment]
I don’t know what that means.

**Comment by wu-clan** [comment]
The generated file path information is printed on the terminal when the test report is generated，sorry, I was misled by IDE.
